introduction/hardware-options/connection-diagrams.md

```diff
@@ -0,0 +1,17 @@
+# Connection Diagrams
+
+The following connection diagrams show how to connect devices to the ArduSub control system. 
+
+Power distribution is not shown other than the power requirements for each device so as to illustrate clean connections. Where a voltage is not known, a "+Vin" is shown. The maximum voltage should be in accordance with the manufacturers specifications.
+
+For more details on connections, the manufacturer's product pages and documentation should be referenced.
+
+## Minimum Required and Recommended Hardware
+
+<img src="/images/hardware/Connection-Diagram-R1.png" class="img-responsive img-center" style="max-height:600px;">
+
+* Right click and 'Save image as...' for the full resolution image.
```

Comment:
You can change the image html to open the new image in full resolution in a new tab:
`<img src="../../images/hardware/Connection-Diagram-R1.png" class="img-responsive img-center" style="max-height:600px;" onclick="window.open('../../images/hardware/Connection-Diagram-R1.png', '_blank')">`

And avoid the right click text

Reply:
I think I fixed this from what Willian gave me too. If its broken when it gets meged, we can revisit.